test/unittests/t_util.cc

```diff
@@ -32,7 +38,94 @@ class ThreadDummy {
 };
 
 
-TEST(T_Util, ThreadProxy) {
+class T_Util : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    const bool retval = MkdirDeep(string(sandbox), 0700);
```

Comment:
It would be better to use CreateTempDir here for the sandbox, which ensures a fresh directory is used